src/styles/sass/main.scss

```diff
@@ -128,19 +128,33 @@ a {
         position: relative;
         text-align: left;
         right: 0;
+        margin: 0 65vw 0 0;
 
         a[rel="up"],
         .icon,
         span, {
           //display: none;
         }
+
+        @media (min-width: 30rem) {
+          margin-right: 25vw;
+        }
+
+        @media (min-width: 60rem) {
+          margin-right: 60vw;
+        }
+
+        @media (min-width: 87.5rem) {
+          margin-right: 73vw;
+        }
       }
 
       @media screen and (min-width: 30rem) {
         > li:nth-child(1) {
           background-color: $nypl-white;
           display: inline-block;
-          margin: 0.175rem 0 0 0.625rem;
+          margin-top: 0.175rem;
```

Comment:
I needed the layout to work with or without the contents button since it won't be there for all the books, so I moved it so the back button has a right margin instead of the contents having a left margin. This was part of moving those margins, I think the 0s still get set elsewhere.